PHPCompatibility/Sniffs/ParameterValues/RemovedSplAutoloadRegisterThrowFalseSniff.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * PHPCompatibility, an external standard for PHP_CodeSniffer.
+ *
+ * @package   PHPCompatibility
+ * @copyright 2012-2020 PHPCompatibility Contributors
+ * @license   https://opensource.org/licenses/LGPL-3.0 LGPL3
+ * @link      https://github.com/PHPCompatibility/PHPCompatibility
+ */
+
+namespace PHPCompatibility\Sniffs\ParameterValues;
+
+use PHPCompatibility\AbstractFunctionCallParameterSniff;
+use PHP_CodeSniffer_File as File;
+
+/**
+ * Detect: Passing `false` to `spl_autoload_register()` is deprecated as of PHP 8.0.
+ *
+ * > spl_autoload_register() will now always throw a TypeError on invalid
+ * > arguments, therefore the second argument $throw is ignored and a
+ * > notice will be emitted if it is set to false.
+ *
+ * PHP version 8.0
+ *
+ * @link https://github.com/php/php-src/blob/c0172aa2bdb9ea223c8491bdb300995b93a857a0/UPGRADING#L393-L395
+ *
+ * @since 10.0.0
+ */
+class RemovedSplAutoloadRegisterThrowFalseSniff extends AbstractFunctionCallParameterSniff
```

Comment:
The classname makes it seem as if it throws false ?

Reply:
Got a suggestion for a better name which still aligns with the naming scheme of these type of ParameterValue sniffs ?